DataFormats/HGCalReco/src/classes_def.xml

```diff
@@ -1,6 +1,8 @@
 
 <lcgdict>
-  <class name="ticl::Trackster" ClassVersion="10">
+  <class name="ticl::Trackster" ClassVersion="12">
```

Comment:
Here and other places we iterating the ClassVersion by two instead of one.  Is there a reason to do that?

Reply:
no, just multiple developments on top of each other. I'll increase only by one